appveyor.yml

```diff
@@ -36,7 +36,7 @@ install:
   - "powershell ci-helpers/appveyor/install-miniconda.ps1"
   - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
   - "activate test"
-  - "conda install numpy scipy"
+  - "conda install numpy'<=1.14' scipy'<=0.18.1'"
```

Comment:
I'll try to fix appveyor in master for mne-bids 0.4 --> so that we can use the env.yml there as well.